manifests/schema.pp

```diff
@@ -49,9 +49,27 @@
   $permissions              = {},
   $tables                   = {},
   $users                    = {},
+  $use_scl                  = $::cassandra::params::use_scl,
+  $scl_name                 = $::cassandra::params::scl_name,
   ) inherits cassandra::params {
   require '::cassandra'
 
+  # Test that the SCL is valid
+  if $use_scl {
+    exec { "test ${scl_name} SCL":
+      command => "/usr/bin/scl enable ${scl_name} \"echo test\"",
```

Comment:
will the exec be executed during each puppet run? I don't think that this is optimal or even required?

Reply:
The exec would indeed have run every time. I have removed this.